src/PeakFinder.hpp

```diff
@@ -0,0 +1,86 @@
+#ifndef SHASTA_PEAKFINDER_HPP
+#define SHASTA_PEAKFINDER_HPP
+
+
+#include <experimental/filesystem>
+#include <stdexcept>
+#include <iostream>
+#include <utility>
```

Comment:
Don't `#include` `<utility>` and `<vector>`. Include the Shasta versions `"utility.hpp"` and `"vector.hpp"` instead, which also inject some `std` names in the `shasta` namespace.